doc/examples/plot_entropy2.py

```diff
@@ -0,0 +1,38 @@
+"""
+=======
+Entropy
+=======
+
+The following example applies the local entropy measure to a noised image with a variable noise.
```

Comment:
I think it would be nice to explain a little more what the example is doing. Please see the other examples for the level of explanations that is needed (altough the existing example on entropy almost does not have any explanations).